controllers/v1/surveysController.js

```diff
@@ -714,43 +703,72 @@ module.exports = class Surveys extends Abstract {
     * @apiUse errorBody
     */
 
-    /**
-      * List of surveys and targetted ones.
-      * @method
-      * @name getSurvey
-      * @param {Object} req - request data.
-      * @returns {JSON} List of surveys with targetted ones.
-     */
+  /**
+   * List of surveys and targetted ones.
+   * @method
+   * @name getSurvey
+   * @param {Object} req - request data.
+   * @returns {JSON} List of surveys with targetted ones.
+   */
 
-    async getSurvey(req) {
-        return new Promise(async (resolve, reject) => {
-            try {
-
-                let surveys = await surveysHelper.getSurvey(
-                    req.body,
-                    req.userDetails.userId,
-                    req.userDetails.userToken,
-                    req.pageSize,
-                    req.pageNo,
-                    req.searchText
-                );
-
-                return resolve({
-                    message: surveys.message,
-                    result: surveys.data
-                });
-
-            } catch (error) {
-                return reject({
-                    status: error.status || httpStatusCode.internal_server_error.status,
-                    message: error.message || httpStatusCode.internal_server_error.message,
-                    errorObject: error
-                });
-            }
-        })
-    }
+  async getSurvey(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let surveys = await surveysHelper.getSurvey(
+          req.body,
+          req.userDetails.userId,
+          req.userDetails.userToken,
+          req.pageSize,
+          req.pageNo,
+          req.searchText
+        );
+
+        return resolve({
+          message: surveys.message,
+          result: surveys.data,
+        });
+      } catch (error) {
+        return reject({
+          status: error.status || httpStatusCode.internal_server_error.status,
+          message:
+            error.message || httpStatusCode.internal_server_error.message,
+          errorObject: error,
+        });
+      }
+    });
+  }
+
+  /**
+   * List of surveys based on user.
+   * @method
+   * @name listSurveyByUserId
+   * @param {Object} req - request data.
+   * @returns {JSON} List of surveys based on user.
+   */
+
+  async listSurveyByUserId(req) {
```

Comment:
controller can be renamed as userSurvey @praveenKDass

Reply:
Same api should be used for big numbers as well

Reply:
Renamed and yes same API using for big number @ankit1shahu